client/Components/createMatch.jsx

```diff
@@ -1,124 +1,169 @@
+                   <h5>Finish Date:</h5>
+                  <input type="date" name="finish" id="finishDate" />
+              </div>
+
+              <div className="four columns">
+                <h5>Type of Match:</h5>
+                <RadioButtonGroup name="shipSpeed" defaultSelected="light"
+                  onChange={function (change, event) {
+                    matchType = event;
+                  }}>
+                   <RadioButton
+                     value="solo"
+                     label="Solo"
+                     style={{marginBottom:16}} />
+                   <RadioButton
+                     value="head"
+                     label="Head-To-Head"
+                     style={{marginBottom:16}}/>
+                </RadioButtonGroup>
+              </div>
+
+
+              <div className="three columns">
+                <h5>Start Funds:</h5>
+                  <DropDownMenu menuItems={menuItems} 
+                    onChange={function (event, index, menuItem) {
+                      startFunds = menuItem.payload;
+                    }} />
+              </div>
+
+            </div>
+
+          </CardText>
+
+           <CardActions expandable={true}>
+            <div className='centerButton'>
+               
+               <RaisedButton label="Submit"
+                
+                linkButton={true} onClick={function(){ 
+                  //date values
+                  let start = document.getElementById('startDate').value;
+                  let end = document.getElementById('finishDate').value;
+
+                  //time formats
+                  let dStart = start.split('-');
+                  let dEnd = end.split('-');
+
+                  let yearStart = dStart[0];
+                  let monthStart = dStart[1]-1;
+                  let dateStart = dStart[2];
+                  let dateIntegerStart = Date.UTC(yearStart,monthStart,dateStart, 14);
+                  let dateFormatStart = new Date(dateIntegerStart);
+
+                  let yearEnd = dEnd[0];
+                  let monthEnd = dEnd[1]-1;
+                  let dateEnd = dEnd[2];
+                  let dateIntegerEnd = Date.UTC(yearEnd,monthEnd,dateEnd, 14);
+                  let dateFormatEnd = new Date(dateIntegerEnd);
+
+                  if (dateFormatStart.toString().substr(0, 3) === 'Sun' || dateFormatStart.toString().substr(0, 3) === 'Sat' ) {
+
+                    return alert('Stock market\'s not open on '+dateFormatStart.toString().substr(0,3) + 'day.');
+                  
+                  } else {
+
+                    if (dateIntegerStart < Date.now() || dateIntegerStart > dateIntegerEnd) {
+                      alert('Matches should not start before today, and should not end before they start.');
+                    } else {
+                      if (!matchType || !startFunds || !dateIntegerStart || !dateIntegerEnd) {
+                        alert('Please set both match type and starting funds for this match.')
```

Comment:
This message should include a section about setting start/end date, too. (even though we will not be keeping the window.alert, let's get the message in there for now anyway)